src/tests/components.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, fireEvent, waitForElementToBeRemoved } from '@testing-library/react';
+
+/* eslint-disable */
+import Home from '../components/home'; 
```

Comment:
- [ ] Kindly make sure all your test cases are passing. as you may notice the `Home` wasn't imported correctly. please make sure you do do.
![image](https://user-images.githubusercontent.com/97100091/228643114-92cc10ef-56b4-4f12-9935-64bcbe066a4c.png)

```suggestion
import Home from '../components/Home'; 
```

Reply:
thanks! its fixed now 🔧